mission/functions/core/ui/taskroster/fn_tr_characterInfo_ribbon_enter.sqf

```diff
@@ -21,13 +21,11 @@
 params["_ctrl_src","_isRibbon"];
 //set description text
 _text = _ctrl_src getVariable ["text",""];
-VN_TR_CHARINFO_REWARD_TEXT_CTRL ctrlSetStructuredText parseText format["<t size='0.65' >%1</t>", _text];
+VN_TR_PLAYERINFO_REWARD_TEXT_CTRL ctrlSetStructuredText parseText format["<t size='0.65' >%1</t>", _text];
 //set img
-_ctrl_preview = VN_TR_CHARINFO_MEDAL_RIBBON_CTRL;
+_ctrl_preview = VN_TR_PLAYERINFO_MEDAL_RIBBON_CTRL;
 _selected_medal = _ctrl_src getVariable ["medal",""];
 _ctrl_preview ctrlSetText _selected_medal;
 
 //sizeCheck currently disabled.
-_ctrl_preview ctrlSetPositionY UIH(15.4);
-_ctrl_preview ctrlSetPositionH UIH(5.66);
```

Comment:
moving position after it was set in the config :/ was not ideal when I edited this thing and then the medal preview element was not moving!